webpack.config.js

```diff
@@ -0,0 +1,15 @@
+/* eslint no-use-before-define: 0 */
```

Comment:
This apparently didn't do much to stop eslint from failing on it in the CI build :/